code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -42,6 +42,10 @@
 			mutantears = /obj/item/organ/internal/ears
 	return ..()
 
+/datum/species/human/felinid/randomize_features(mob/living/carbon/human/human_mob)
+	randomize_external_organs(human_mob)
+	..()
```

Comment:
```suggestion
	return ..()
```

Reply:
Though this isn't called anywhere else, and the base proc is just a return. Do we WANT to call ``..()``?

Reply:
felinid species is a human species subtype and uses skin colour, so it's calling the /datum/species/human/randomize_features() parent for that